### ISSUE TITLE
The line breaks inside airtable entries don't exist

### ISSUE THREAD

**Issue by malikpiara** [upframe/internships#11]
Airtable doesn't seem to be handling the line breaks we input manually. Take a look at https://internships.upframe.io/uniplaces/operations-ambassadors-spain

This is an important priority, can we fix it? @ulissesferreira @fabiofcferreira

**Comment by malikpiara** [comment]
How are we here @fabiofcferreira? Can you help @ulissesferreira?

**Comment by ulissesferreira** [comment]
Yes I can I am on it now. We are going to use a special character just for that. Then we try it out and we can choose any character we want

**Comment by ulissesferreira** [comment]
I'm gonna try something now
